src/components/app.tsx

```diff
@@ -41,7 +41,18 @@ export const AppComponent = observer(function WrappedComponent() {
         </div>
       }
       <div className={css.timeDisplay}>
-        {timeInDays} {timeInDays === 1 ? "day" : "days"} and <br /> {timeHours} {timeHours === 1 ? "hour" : "hours"}
+        {
+          simulation.isFireEventActive &&
+          <>
+            {timeInDays} {timeInDays === 1 ? "day" : "days"} and {timeHours} {timeHours === 1 ? "hour" : "hours"}
+          </>
+        }
+        {
+          !simulation.isFireEventActive &&
+          <>
+            {timeInYears} years
```

Comment:
Do you need to handle the single year case for pluralization like in the days/hours code above?

Reply:
Nevermind, I see you removed the code in the next PR (https://github.com/concord-consortium/forestfire/pull/7)